pkg/azure/resources.go

```diff
@@ -55,6 +55,10 @@ func getClusterUri(subscriptionId string, resourceGroupName string, clusterName
 	return fmt.Sprintf("/subscriptions/%s/resourceGroups/%s/providers/%s/%s", subscriptionId, resourceGroupName, managedClusterTypeName, clusterName)
 }
 
+func getDeploymentUri(subscriptionId string, resourceGroupName string) string {
+	return fmt.Sprintf("/subscriptions/%s/resourceGroups/%s", subscriptionId, resourceGroupName)
+}
+
```

Comment:
This might be better placed in naming.go?

Reply:
Sounds good. Updated